### DISCUSSION TITLE
Poor availability of screens with supported chipsets - support for new one?

### DISCUSSION THREAD

**Discussion by baniaque** [Bodmer/TFT_eSPI#1165] [New processor/display support or capability request]
Hi,

I appreciate the huge amount of work you put into the library. I use it in every project with LCD screen. For some time it has been getting harder and harder to buy 4-inch screens with chips that are supported by your library. It looks like the end of production to me, only some bleeds remain. Instead of them, screens with higher resolution (800 * 480) and with other systems appeared on the market - NT35510, OTM8009A etc.
Are you considering adding support for these screens? Currently, the lack of availability of screens has stopped me, and I'm worried.

best raegards

**Comment by Bodmer** [comment]
Yes, the cheap screens are in decline as suppliers move to more profitable large screens.  I am busy with other projects and do not have one of those screens to debug the driver.
